webpack.config.js

```diff
@@ -1,45 +1,37 @@
-var path = require(`path`);
-var ExtractTextPlugin = require(`extract-text-webpack-plugin`);
 var webpack = require(`webpack`);
 
+var plugins = [];
+
+if (process.env.NODE_ENV === `production`) {
+  console.log(`bundling for production.`);
+  plugins = [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(`production`)
+      }
+    }),
+    new webpack.optimize.UglifyJsPlugin()
+  ];
+}
+
 module.exports = {
+  context: `${__dirname}`,
   entry: `./client.jsx`,
-
   output: {
-    filename: `[name].js`,
-    chunkFilename: `[id].chunk.js`,
-    path: path.join(`public`, `build`),
+    path: `${__dirname}/public/build`,
+    filename: `bundle.js`,
     publicPath: `/build/`
   },
-
   module: {
-    loaders: [
+    rules: [
       {
-        test: /\.js(x?)$/,
+        test: /.jsx?$/,
         exclude: /node_modules/,
-        loader: `babel`,
-        query: {
-          presets: [`es2015`, `react`]
-        }
-      },
-      { test: /\.json$/, loaders: [`json-loader`] },
-      {
-        test: /\.css$/,
```

Comment:
These were required by `react-fa`: https://www.npmjs.com/package/react-fa#webpack-setup Since we don't use `react-fa` anymore, we can remove this chunk.